docs/VECTORLINT.md

```
@@ -0,0 +1,77 @@
+# VectorLint example styling rules
+
+{/* This file is prepended to the system prompt for every VectorLint evaluation 
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)